### PR TITLE
typos

### PR DESCRIPTION
**Pull request by Fourchaux**
Typos found with https://github.com/codespell-project/codespell

### CODE REVIEW

File: release-info/howto.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ git push --tags
 
 ## 5-bis: Alternative for branching
 
-This needs to be more tested, tread with care.
+This needs to be more tested, treat with care.
```
Comment (dra27) [inline release-info/howto.md]: Ahem, please see thread at https://github.com/ocaml/ocaml/pull/9162#discussion_r353779455

**Comment by dra27** [comment]
All good, apart from the repeated one rejected in #9162, thanks!

File: release-info/howto.md (hunk 1 above)
Comment (Fourchaux) [reply release-info/howto.md]: Damn,damn...

**Comment by dra27** [comment]
Thanks!

File: release-info/howto.md (hunk 1 above)
Comment (xavierleroy) [reply release-info/howto.md]: When editing documentation and source comments, "maybe your best course is to tread lightly".  (Context: https://mashable.com/article/breaking-bad-bryan-cranston-quote)

**Pull request merged**